contrib/check-config.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 if ! command -v zgrep &> /dev/null; then
 	zgrep() {
-		zcat "$2" | grep "$1"
+		zcat "${!#}" | grep "${@:1:$#-1}"
```

Comment:
This is pretty gross.

Reply:
This is a bash file so that is as elegant as it gets for "last parameter" and "all but the last parameter"

Reply:
What was wrong with `$2` and `$1`?

Reply:
zgrep supports many flags that are passed on to grep. If we are going to re-implement zgrep it should be done properly and pass flags. Specifically in this case we are passing "-E". Sure we could force -E grep and not pass more than one parameter but that's not robust/ elegant/etc.